common/lib/dependabot/clients/codecommit.rb

```diff
@@ -68,8 +68,8 @@ def fetch_file_contents(repo, commit, path)
           commit_specifier: commit,
           file_path: path
         ).file_content
-        rescue Aws::CodeCommit::Errors::FileDoesNotExistException
-          raise NotFound
+      rescue Aws::CodeCommit::Errors::FileDoesNotExistException
+        raise NotFound
```

Comment:
Should this get pulled out to a separate PR? I realize it's trivial but I'll be merging via squash so if for some reason we have to revert I'd rather not revert this change as it's unrelated to truncation AFAICT

Reply:
I see this resolved, but I don't see a separate PR for it? Don't want to lose track of it if it's something that needs doing...

Reply:
I can add one in, I just haven't had the time, yet.  I hit this as time allows. I don't think it is high priority as it has not been causing issues (that I know of) the linter was just complaining about it, I don't think it needs to hold this one up.